RELEASE.md

```diff
@@ -0,0 +1,120 @@
+
+If releasing a new version 5.4.0 with the latest release being 5.3.8, for
+example, they will look as follows:
+
+    export MAJOR=5
+    export MINOR=4
+    export PATCH=0
+    export LAST_RELEASE=5.3.8
+
+## Create release prep branch
+
+This branch will be used to prepare all the necessary things to get ready for
+release.
+
+    git checkout -b v$MAJOR.$MINOR.$PATCH-prep
+
+## Prepare the release notes
+
+Using https://github.com/cilium/release, prepare the release notes between the
+last minor version (latest patch) and current.
+
+    ./release --repo cilium/cilium-cli --base v$LAST_RELEASE --head master
+    **Other Changes:**
+    * install: Add a hidden --base-version flag (#418, @michi-covalent)
+    * Makefile: introduce GO_BUILD variable (#432, @tklauser)
+    * Prepare for release v0.8.5 (#428, @michi-covalent)
+    * Run "Post-test information gathering" step on cancellation (#426, @michi-covalent)
+    * skip Succeeded pods (#431, @xyz-li)
+    ... etc ...
+
+## Update the `VERSION` file for the new release
+
+Usually this only consists of dropping the `-dev` suffix from the string.
+
+Commit the changes to the prep branch:
+
+    git add VERSION
+    git commit -s -m "Prepare for release v$MAJOR.$MINOR.$PATCH"
+
+## Update the `VERSION` file for next development cycle
+
+Usually this only consists of bumping the patch version.
+
+Commit the changes to the prep branch:
+
+    git add VERSION
+    git commit -s -m "Prepare for v$MAJOR.$MINOR.<$PATCH+1> development"
+
+## Push the prep branch and open a Pull Request
+
+The pull request has to be `v$MAJOR.$MINOR.$PATCH-prep -> master`
+
+Once the pull request is approved and merged, a tag can be created.
+
+## Tag a release
+
+Identify the right commit and tag the release. Usually, the commit modifying
+the version in the VERSION file is tagged.
+
+Example:
+
+    git tag -a v0.8.5 -m 'v0.8.5' <commit-sha>
+
+Then push the tag.
+
+Example:
+
+    git push origin v0.8.5
+
+## Update the GitHub release notes
+
+When a tag is pushed, a GitHub Action job takes care of creating a new GitHub
+draft release, building artifacts and attaching them to the draft release.
+
+The release notes need to be manually added before manually publishing the
+release.
+
+## (OPTIONAL) Update `stable.txt` in the master branch
+
+Hubble's installation instruction in the Cilium documentation uses the version specified in
+`stable.txt` in the master branch. There are a couple of things to consider when deciding
+whether to update `stable.txt`. Let's say `stable.txt` is currently pointing to `v0.6.1`:
+
+- If this is a minor or patch release relative to the current stable version (i.e. `v0.7.0`
+  or `v0.6.2`), update `stable.txt` so that people start picking up the new features / bug
+  fixes included in this release.
+- If this is a patch release of a previous version (e.g. `v0.5.2`), don't update
+  `stable.txt`.
+- If this is a major release (e.g. `v1.0.0`), the installation instructions in older Cilium
```

Comment:
This is taken straight from Hubble's `RELEASE.md`. I don't think it applies to Cilium CLI.

Reply:
Indeed, thanks. I was a bit to quick to push the PR. Now folded the `stable.txt` update into a section above.